### ISSUE TITLE
"Something went wrong" but no error message is displayed

### ISSUE THREAD

**Issue by gabek** [ruzickap/action-my-broken-link-checker#128]
**Describe the bug**

The action fails with the following:

```
*** ERROR: [2024-02-09 03:38:43] Something went wrong - see the errors above...
```

But no errors are displayed.

Do you have any ideas what might be making it fail, or how I can get more details to fix it?

**To Reproduce**

Run the action with the following settings:

```
with:
    url: https://owncast.online/docs
    cmd_params: --buffer-size=8192 --max-connections=10 --color=always --rate-limit=1 --max-connections-per-host 3 --exclude="opencollective.com|github.com|example.com|mydomain.com|sitemap" --verbose
```

**Comment by ruzickap** [comment]
Can you paste here the whole step to allow me to see all the parameters "with:" + "uses:" please?

**Comment by gabek** [comment]
Sure!  Thanks!

```yaml
runs-on: ubuntu-latest
    steps:
      - name: Check broken links
        uses: ruzickap/action-my-broken-link-checker@v2
        with:
          url: https://owncast.online/docs
          cmd_params: '--buffer-size=8192 --max-connections=10 --color=always --rate-limit=1 --max-connections-per-host 3 --exclude="opencollective.com|github.com|example.com|mydomain.com|sitemap" --verbose'
```

**Comment by ruzickap** [comment]
I can see some results when running it in my test repo: https://github.com/ruzickap/gha-test/actions/runs/7841180518/job/21397059636

Test GitHub Action: https://github.com/ruzickap/gha-test/actions/runs/7841180518/workflow

Maybe try to add `debug: true` to get more details + commands like I did in my [test](https://github.com/ruzickap/gha-test/blob/9e7a42d5f3193b76aafde211c1f812ea047da5fc/.github/workflows/test.yml#L19).

---

Anyway - I would not recommend you to use the `action-my-broken-link-checker` for checking the external sites.

There are better projects which can do it like https://github.com/lycheeverse/lychee-action which suits better for this use case.

I personally using [lychee](https://github.com/lycheeverse/lychee) everywhere I can, because it has more active community than [muffet](https://github.com/raviqqe/muffet) (which is being used in `action-my-broken-link-checker`).

**Comment by gabek** [comment]
The downside about the Lychee action is it creates one huge issue with all of the broken links instead of one issue per link, so that's why I was trying to get this one to work.

I changed my configuration to point to a local directory and use the Caddy approach, but it fails in the exact same way. A generic message with no actionable error when `error_trap` fires.

I'll just have to keep looking for a solution. Thanks for responding.
